packages/jsii/lib/symbol-id.ts

```diff
@@ -6,6 +6,11 @@ export function symbolIdentifier(
   typeChecker: ts.TypeChecker,
   sym: ts.Symbol,
 ): string | undefined {
+  // If this symbol happens to be an alias, resolve it first
+  if ((sym.flags & ts.SymbolFlags.Alias) !== 0) {
+    sym = typeChecker.getAliasedSymbol(sym);
+  }
```

Comment:
Is it maybe possible that an aliased symbol refers to another symbol that is itself an alias? Or does the `typeChecker.getAliasedSymbol` resolve all the way out?

Reply:
Good question. I'll turn it into a `while` loop just to be safe.